embarkConfig/contracts.js

```diff
@@ -227,19 +181,7 @@ module.exports = {
     ],
     strategy: 'explicit',
```

Comment:
I see `strategy` is being put in every config, we con only define it in `default` if we want